docs/elements/hx-tile/index.html

```diff
@@ -0,0 +1,34 @@
+---
+title: <hx-tile>
+also:
+  components/choice-tiles: Choice Tiles
+  elements/hx-tile-title: <hx-tile-title>
+  elements/hx-tile-description: <hx-tile-description>
+---
+{% extends 'element.njk' %}
+{% block content %}
+  <section>
+    <p>
+      The custom <code>&lt;hx-tile&gt;</code> element is compromised of an icon,
+      label, and optional description.
+    </p>
+
+     <hx-dl class="hxBox-md metadata">
+      <hx-def>
+        <hx-dt>Permitted Parents</hx-dt>
+        <hx-dd>any</hx-dd>
+      </hx-def>
+      <hx-def>
+        <hx-dt>Permitted Children</hx-dt>
+        <hx-dd>
+          in this order:
+          <ol>
+            <li>a required <a href="elements/hx-icon">&lt;hx-icon&gt;</a> element</li>
```

Comment:
This may not be 100% true. The tile icon requires a div wrapper with a specific class. However, this can be updated to specify the "checkmark" icon, specifically.

Reply:
We may need to do a fast follow to create an `<hx-tile-icon>` to wrap an `<hx-icon>`.

Reply:
You will need to specify that the `div.hx-tile-icon` is needed to display the tile icon (not the checkmark).